tests/CMakeLists.txt

```diff
@@ -259,6 +259,11 @@ endif()
 
 target_link_libraries(pmemkv_test pmemkv libgtest ${CMAKE_DL_LIBS})
 
+# It should be removed when all supported distributions update gtest
```

Comment:
please add TODO or XXX mark here, please

and can you specify which min. gtest version we're looking for?